packages/sanes-chrome-extension/package.json

```diff
@@ -3,23 +3,31 @@
     "use-config-development": "cp ./config/development.json ./public/assets/config/conf.json",
     "use-config-production": "cp ./config/production.json ./public/assets/config/conf.json",
-    "build": "react-scripts build src/extension/contentscript.ts src/extension/backgroundscript.ts",
-    "export": "yarn use-config-production && yarn build && web-ext build --source-dir ./build/ --artifacts-dir ./exports --overwrite-dest",
+    "build-with-any-config": "react-scripts build src/extension/contentscript.ts src/extension/backgroundscript.ts",
+    "postbuild-with-any-config": "./scripts/set_manifest_version.sh",
+    "build-test": "yarn use-config-test && yarn build-with-any-config",
+    "build-development": "yarn use-config-development && yarn build-with-any-config",
+    "build-production": "yarn use-config-production && yarn build-with-any-config",
+    "build": "yarn build-test",
+    "override-config-test": "cp ./config/test.json ./build/assets/config/conf.json",
+    "override-config-development": "cp ./config/development.json ./build/assets/config/conf.json",
+    "override-config-production": "cp ./config/production.json ./build/assets/config/conf.json",
+    "export-development": "./scripts/remove_key.sh && yarn override-config-development && web-ext build --source-dir ./build/ --artifacts-dir ./exports --overwrite-dest",
+    "export-production": "./scripts/remove_key.sh && yarn override-config-production && web-ext build --source-dir ./build/ --artifacts-dir ./exports --overwrite-dest",
     "format": "prettier --write --loglevel warn './src/**/*.{ts,tsx,json,md,css}'",
     "lint": "eslint -c .eslintrc.js --max-warnings 0 'src/**/*.ts{,x}'",
     "start": "yarn use-config-test && react-scripts start src/extension/contentscript.ts src/extension/backgroundscript.ts",
-    "start:firefox:test": "yarn use-config-test && yarn build && web-ext run --source-dir ./build/",
-    "start:firefox:development": "yarn use-config-development && yarn build && web-ext run --source-dir ./build/",
-    "start:firefox:production": "yarn use-config-production && yarn build && web-ext run --source-dir ./build/",
+    "start:firefox:test": "yarn build-test && web-ext run --source-dir ./build/",
+    "start:firefox:development": "yarn build-development && web-ext run --source-dir ./build/",
+    "start:firefox:production": "yarn build-production && web-ext run --source-dir ./build/",
     "test-dom": "yarn use-config-test && CI=true react-scripts test '\\.dom\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom",
     "test-e2e": "yarn use-config-test && CI=true react-scripts test '\\.e2e\\.(spec|test)\\.[jt]s[x]?$' --runInBand --env=jsdom",
     "test-unit": "yarn use-config-test && CI=true react-scripts test '\\.unit\\.(spec|test)\\.[jt]s[x]?$' --env=jsdom",
     "test-single": "yarn use-config-test && CI=true react-scripts test --env=jsdom",
-    "test": "yarn use-config-travis && yarn test-unit && yarn test-dom && yarn test-e2e"
```

Comment:
These are needed for adding config files after a clean build, otherwise, it will fail.

Reply:
The the config file is written into the build folder in `build-test` script now, which is called in a clean `yarn build`. So it is always there, no matter if you test or not.